posts/helpers.py

```diff
@@ -0,0 +1,36 @@
+from .models import *
+
+def get_user( pk):
+        try:
+            return User.objects.get(pk=pk)
+        except User.DoesNotExist:
```

Comment:
This is worry some I will look into it

Reply:
Think this should be a global exception incase an invalid uuid is passed. Also we should return false and let the caller deal with the not found behavior